sound/soc/intel/boards/sof_maxim_common.c

```diff
@@ -12,6 +12,20 @@
 #include <uapi/sound/asound.h>
 #include "sof_maxim_common.h"
 
+/*
+ * Common structures and functions
+ */
+static const struct snd_kcontrol_new maxim_2spk_kcontrols[] = {
+	SOC_DAPM_PIN_SWITCH("Left Spk"),
+	SOC_DAPM_PIN_SWITCH("Right Spk"),
+
+};
```

Comment:
commit title prefix should be ASoC: Intel: move rt5682/nau8825 speaker widget to common modules

Reply:
done